mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -1093,6 +1093,27 @@ private[python] class PythonMLLibAPI extends Serializable {
     LinearDataGenerator.generateLinearRDD(
       sc, nexamples, nfeatures, eps, nparts, intercept)
   }
+
+  /**
+   * Wrapper around Statistics.kolmogorovSmirnovTestWrapper with default params.
+   */
+  def kolmogorovSmirnovTestWrapper(
+      data: JavaRDD[Double],
+      distName: String): KolmogorovSmirnovTestResult = {
+    Statistics.kolmogorovSmirnovTest(data, distName)
+  }
+
+  /**
+   * Wrapper around Statistics.kolmogorovSmirnovTestWrapper.
+   */
+  def kolmogorovSmirnovTestWrapper(
+      data: JavaRDD[Double],
+      distName: String,
+      params: JList[Double]): KolmogorovSmirnovTestResult = {
+    val seqParams = params.asScala.toSeq
+    Statistics.kolmogorovSmirnovTest(data, distName, seqParams: _*)
+  }
+
```

Comment:
+1 on only one wrapper. And we don't need to call the method `Wrapper`. Just follow how `chiSqTest` is implemented.